core/src/main/scala/spinal/core/Misc.scala

```diff
@@ -49,6 +49,26 @@ object log2Up {
   def apply(value : Int) : Int = apply(BigInt(value))
 }
 
+object Gray {
+  /**
+   * Encoding binary number in binary gray code
+   */
```

Comment:
```suggestion
  /** Encoding binary number in binary gray code */
```

Reply:
changed, but these are now the only two one-line comments formatted this way in this file